tests/requirements.in

```diff
@@ -3,6 +3,8 @@
 ## Database Drivers
 asyncpg
 aiomysql
+aioodbc
+pyodbc
```

Comment:
`aioodbc` already requires `pyodbc`, So please leave it out as `pip-tools` already resolves it for us.
Only specify a secondary dependancy if there is a constraint you need to apply to it.

Reply:
Will do. FYI, I will be a bit less active this week. I have a week off for home renovations prior to an appraisal. : / 

Time constraints aside, I am currently trying to install Oracle XE on my personal so as to figure out how to do testing.

Thank you!

Reply:
Oh, selling? I definitely get better deals when I can honestly say "everything you see is working"
Upgrades didn't help, but a very clean place & a good photographer definitely did.

I technically have enough outstanding tasks to keep me busy for weeks, but prefer exciting features over bugfixes. Alas!

Reply:
Haha, no just a refinance. But it sounds like we have the same issues with house work. Too many cool projects to be bothered with all the boring detail stuffs.